datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/cliUtils/CliParametersParser.scala

```diff
@@ -104,20 +104,25 @@ object CliParametersParser {
     getLoadOptions(finalOutMapWithDefaults, outputPrefix)
   }
 
-  private[cliUtils] def genericTrioParse(args: Array[String]): (Option[String], Set[String], OptionsTrio) = {
+  private[cliUtils] def genericTrioParse(args: Array[String]): (Option[String], Set[String], String, OptionsTrio) = {
```

Comment:
Not a trio anymore

Reply:
Sure. I have missed it.

Reply:
Method renamed to "genericQuartettoParse" in 0da1a40805f0860d784ee4133fdbd2f57b51ff07.